students/tri_nguyen/lesson08/test_Circle_class.py

```diff
@@ -0,0 +1,108 @@
+# ------------------------------------------- #
+# Student:   Tri Nguyen
+# Instructor: Natasha Aleksandrova
+# Date:  01-Apr-2018
+# ------------------------------------------- #
+
+import unittest
+from Circle_class import Circle
```

Comment:
per pep8, modules should be lower case:
`from circle_class import Circle` and _class shouldn't really be part of it in real scenarios -> `from circle import Circle`